python/ray/tune/trainable.py

```diff
@@ -349,9 +359,13 @@ def stop(self):
         self._result_logger.close()
         self._stop()
 
-    def _train(self):
+    def _train(self, evaluate=False):
```

Comment:
Defaulting to true is more backwards compatible.

Reply:
In this PR, `evaluate` should only be True if you're running evaluation and not training

Reply:
Unless you're suggesting `def _train(self, update=True)`